plugins/k8saudit-eks/Makefile

```diff
@@ -0,0 +1,36 @@
+#
```

Comment:
The makefile of other plugins have been changed recently to support automatic readme generation. See https://github.com/falcosecurity/plugins/pull/138. I would be great if we could have those changes here as well.

For the future: maybe a makefile template (at least for Go plugins) might be the way to go.

Reply:
:+1:

Reply:
It's updated